seong-geun/app.js

```diff
@@ -115,22 +129,55 @@ app.get("/u_posts", async (req, res) => {
    `,
     [userId]
   );
+
   res.status(200).json({ data: userspost });
-  const { id, profile_image, username, age, email } = req.body;
+});
 
+// 게시글 수정하기
+app.patch("/m_posts", async (req, res) => {
+  const { postingUserId, postingContent } = req.body;
   await appDataSource.query(
-    `INSERT INTO users(
-      id,
-      profile_image,
-      username,
-      age,
-      email
-		) VALUES (?, ?, ?, ?, ?);
-	`,
-    [id, profile_image, username, age, email]
+    `UPDATE
+        posts
+    SET
+        posting_content = ?
+    WHERE
+        posts.user_id = ? 
+    `,
+    [postingContent, postingUserId]
   );
 
-  res.status(201).json({ message: "userCreated" });
+  const m_userspost = await appDataSource.query(
```

Comment:
```suggestion
  const userspost = await appDataSource.query(
```
javascript에서는 기본적으로 코드 컨벤션 중 snake_case(밑줄로서 단어를 이어나가는) 방법을 사용하지 않습니다. camelCase(소문자로 시작하여 대문자를 섞는 방법)을 활용하거나 하여 수정해주세요